plugins/vis-editor-spriter/src/main/java/com/kotcrab/vis/plugin/spriter/ui/VisSpriterComponentTable.java

```diff
@@ -42,11 +43,17 @@ public VisSpriterComponentTable (ModuleInjector injector) {
 	protected void init () {
 		super.init();
 
+		entitySelectBox = new VisSelectBox<>();
+		entitySelectBox.getSelection().setProgrammaticChangeEvents(false);
+		entitySelectBox.addListener(properties.getSharedSelectBoxChangeListener());
+		
 		animSelectBox = new VisSelectBox<>();
 		animSelectBox.getSelection().setProgrammaticChangeEvents(false);
 		animSelectBox.addListener(properties.getSharedSelectBoxChangeListener());
 
 		VisTable table = new VisTable(true);
+		table.add(new VisLabel("Entity"));
+		table.add(entitySelectBox);
```

Comment:
Add table.row() after `table.add(entitySelectBox);` so UI can fit in Entity Properties panel. 

Also please run our code formatter on code changes (see CONTRIBUTING files).